cognite_toolkit/_cdf_tk/commands/_profile.py

```diff
@@ -67,7 +68,7 @@ class Columns:  # Placeholder for columns, subclasses should define their own Co
     @cached_property
     def columns(self) -> tuple[str, ...]:
         return (
-            tuple([attr for attr in self.Columns.__dict__.keys() if not attr.startswith("_")])
+            tuple([value for attr, value in self.Columns.__dict__.items() if not attr.startswith("_")])
```

Comment:
Related bug, we should use the attribute value and not name as the column heading.